mulighetsrommet-api/src/main/resources/web/openapi.yml

```diff
@@ -20,6 +20,28 @@ paths:
               schema:
                 type: string
                 enum: [PONG]
+  /api/v1/sanity:
+    get:
+      tags:
+        - mulighetsrommet
+      operationId: sanityQuery
+      parameters:
+        - in: query
+          name: query
+          schema:
+            type: string
+          description: Sanity query
```

Comment:
Kan du utvide til `Sanity query as valid Groq-string. See https://www.sanity.io/docs/groq for more information.